data/_models/shared/Contacts.csv

```diff
@@ -13,7 +13,8 @@ Contacts,,title,ontology,,,CatalogueOntologies,Titles,,,,foaf:title,Title of the
 Contacts,,organisation,ref,,,,Organisations,resource,,,foaf:Organization,Affiliated organisation of the contact person,"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts,INTEGRATE",,
 Contacts,,email,email,,,,,,,,"vcard:hasEmail,foaf:mbox",Contact's email address,"DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts,INTEGRATE",,
 Contacts,,statement of consent email,bool,,TRUE,,,,,,,"Indicate whether this person consents to display their email address in the catalogue (which is openly accessible). The consent also includes future linkages with other national and international catalogues and platforms to increase visibility and findability (e.g. BBMRI-ERIC, NARCIS) of UMCG-resources. Contributors without a positive statement of consent for their email address will be shown without their email address in the UMCG Research Data Catalogue",UMCGCohortsStaging,,
-Contacts,,orcid,,,,,,,,,,Orcid of the contact person,"DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts,INTEGRATE",,
-Contacts,,homepage,hyperlink,,,,,,,,"foaf:workplaceHomepage,vcard:hasURL",Link to contact's homepage,"DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,INTEGRATE",,
+Contacts,,orcid,,,,,,,,,dcterms:identifier,Orcid of the contact person,"DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts,INTEGRATE",,
+Contacts,,orcidUrl,,,,,,,,,vcard:url,Orcid URL of the contact person,"DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts,INTEGRATE",,orcid ? 'https://orcid.org/'+orcid : null
```

Comment:
orcid url can be deleted, I think, because according to the metadata schema it is not mandatory, so SHACLs should be changed: https://github.com/Health-RI/health-ri-metadata/blob/develop/Images/2.0_plateau2/HRI_metadata_p2.png